src/Sulu/Bundle/TestBundle/Testing/TestUserProvider.php

```diff
@@ -31,28 +31,53 @@ class TestUserProvider implements UserProviderInterface
      */
     private $user;
 
-    public function __construct(ObjectManager $em)
+    /**
+     * @var EntityManager
+     */
+    private $entityManager;
+
+    /**
+     * @param EntityManager $entityManager
+     */
+    public function __construct(EntityManager $em)
+    {
+        $this->entityManager = $em;
```

Comment:
Should never do logic like creating a new user in the _constructor of a class. Instantiating a class should not cause other things to change.